### ISSUE TITLE
Is the project still alive?

### ISSUE THREAD

**Issue by alexdmz** [OpenDevelopmentMethod/discussion#14]
very interesting thing, since i saw the article on OpenSource.com and watched the presentation i felt in love with, but haven't seen to much follow up to it, is it still in development / discussion?

**Comment by wrupple** [comment]
it's alive as long as there's people discussing it, i suppose.

**Comment by ahmadnassri** [comment]
@alexdmz not at all, I have personally been presenting and talking about this in various events and forums, I owe the website an update, and more content to be added! 

also as @wrupple mentioned, as long as the community is discussing and adding their own feedback and providing direction, then it will continue to grow!
